Sources/SnapshotTesting/PropertyListEncoder.swift

```diff
@@ -0,0 +1,1827 @@
+#if os(Linux)
```

Comment:
Tried making a suggested change but GitHub doesn't seem to let me for new files. Can you remove this guard?